### PR TITLE
Remove roman line numbers

### LINKED ISSUE #2238: Unordered lists of dates render as roman numerals
**Description**

An unordered list of dates is rendered as roman numerals.
Also, autocompletion does not continue the unordered list.

**To Reproduce**

1. Create a list of dates on hedgedoc.dev:
```
Test:
- 1. January
- 2. January
```
3. Dates render as roman numerals:
```
Test:
- i. January
- ii. January
```
4. Press Enter
5. The list is autocompleted as:
```
Test:
- 1. January
- 2. January
  3. 
```

**Expected behavior**

The dates should render unchanged, as a normal unordered list.
The list should autocomplete as an unordered list.

**Screenshots**

**Your Setup (please complete the following information):**
 - OS: [e.g. iOS 13, Ubuntu 20.04, Windows 10]
 - Browser with version [e.g. Chrome 81, Safari 13.1]
   - Firefox 102.0
 - HedgeDoc version (Click "Version info" in the footer)
     - server mock
     - client 0.0.0+94bd3a4d

**Additional context**

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
Markdown stylesheet

### Description
This PR removes the roman numbers for listings.

### Steps

- [x] Added implementation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/react-client/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### Related Issue(s)
Fixes #2238

### CODE REVIEW

**Review by DerMolly — APPROVED** [review]

File: global-styles/github-markdown.scss [hunk 1]
```diff
@@ -73,939 +13,993 @@
...
+  table tr {
+    // background-color: #fff;
+    border-top: 1px solid #c6cbd1;
+  }
 
-.markdown-body pre {
-  word-wrap: normal;
-}
+  table tr:nth-child(2n) {
+    //background-color: #f6f8fa;
```
Comment (ErikMichelson) [inline global-styles/github-markdown.scss:791]: If this is commented out, why not remove it?

File: global-styles/github-markdown.scss [hunk 2]
```diff
@@ -73,939 +13,993 @@
...
-  max-height: 240px;
-  overflow-y: auto;
-}
+  .highlight pre,
+  pre {
+    padding: 16px;
+    overflow: auto;
+    font-size: 85%;
+    line-height: 1.45;
+    //background-color: #f6f8fa;
```
Comment (ErikMichelson) [inline global-styles/github-markdown.scss:845]: Same here

File: global-styles/github-markdown.scss [hunk 3]
```diff
@@ -73,939 +13,993 @@
...
+    cursor: pointer;
+  }
 
-.markdown-body h4 {
-  font-size: 16px;
-}
+  kbd {
+    display: inline-block;
+    padding: 3px 5px;
+    font: 11px SFMono-Regular, Consolas, Liberation Mono, Menlo, monospace;
```
Comment (ErikMichelson) [inline global-styles/github-markdown.scss:193]: Do we want to insert the same monospaced font from the editor here?

File: global-styles/github-markdown.scss [hunk 4]
```diff
@@ -1,69 +1,9 @@
-/*
- * Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (https://sindresorhus.com)
- * SPDX-FileCopyrightText: 2020 Sindre Sorhus <sindresorhus@gmail.com> (https://sindresorhus.com)
+/*!
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
```
Comment (ErikMichelson) [inline global-styles/github-markdown.scss:2]: Is it correct to change this without giving credit to Sindre? The only changeset is the use of scss nesting and the line number fix...

**Review by ErikMichelson — CHANGES_REQUESTED** [review]

File: global-styles/github-markdown.scss (hunk 3 above)
Comment (mrdrogdrog) [reply global-styles/github-markdown.scss:193]: Nope. It's override by another file.

Comment (ErikMichelson) [reply global-styles/github-markdown.scss:193]: Okay

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**